### PR TITLE
hidden entities copies

### PR DESCRIPTION
**Pull request by jakub-rdx**

### CODE REVIEW

File: Content/0.4.5 - Preferences/preferences.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "preferences_title": "Preferences",
     "preferences_depositGuarantees_title": "Default Deposit Guarantees",
     "preferences_depositGuarantees_subtitle": "Set your guaranteed minimum for estimated deposits",
-    "preferences_hiddenEntities_title": "Hidden Accounts and Personas",
-    "preferences_hiddenEntities_subtitle": "Manage hidden Accounts and Personas",
+    "preferences_hiddenEntities_title": "Hidden Personas &amp; Accounts",
```
Comment (matiasbzurovski) [inline Content/0.4.5 - Preferences/preferences.json:5]: why not use `&` directly? We are already doing so for example [here](https://github.com/radixdlt/apps-localization-src/blob/main/Content/0.4%20-%20Settings/settings.json#L5)

Comment (jakub-rdx) [reply Content/0.4.5 - Preferences/preferences.json:5]: Android IDE suggest to use escaped version in strings file on Android, I just done it automatically without giving it a thought. But it looks that crowdin does that conversion, so I will change it to `&`

**Review by sergiupuhalschi-rdx — APPROVED** [review]

File: en/en-US/Content/0.4.5 - App Settings/appSettings.json [hunk 2]
```diff
@@ -3,16 +3,6 @@
 	"appSettings_subtitle": "Customize your Radix Wallet",
 	"appSettings_linkedConnectors_title": "Linked Connectors",
 	"appSettings_gateways_title": "Network Gateways",
-	"appSettings_entityHiding_title": "Account &amp; Persona Hiding",
```
Comment (matiasbzurovski) [inline en/en-US/Content/0.4.5 - App Settings/appSettings.json:6]: are these changes on purpose? We shouldn't be modifying the language specific (in this case, English-US) 
 files ourselves

Comment (jakub-rdx) [reply en/en-US/Content/0.4.5 - App Settings/appSettings.json:6]: my mistake. Reverted this file to how it was
